lib/streamlit/elements/__init__.py

```diff
@@ -41,6 +41,7 @@
     "arrow_line_chart",
     "arrow_table",
     "arrow_vega_lite_chart",
+    "audio",
```

Comment:
should audio and video be put into nonwidget elements?

Reply:
That's where they went. You might have misread the diff, since it was kinda weird